docs/devGuide/bootcamp/contributeToDocs.md

```diff
@@ -0,0 +1,136 @@
+
+</box>
+
+## Make Modifications to the Documentation
+
+In the following steps, we will be fixing a hypothetical typo. You do not need to commit the changes or raise a PR, but understand the context and follow the steps to get a feel for the process.
+
+**!!Let's suppose we want to change the use of "Pop-Up" in our [user guide](../../userGuide/components/popups.html) to "Popup" (i.e. remove the hyphen).!!**
+
+<box type="info" light>
+
+These are the typical motivations for making documentation changes:
+- Fixing typos, grammar, or spelling mistakes when reading the documentation, which may not be code-related.
+- Keeping the documentation up-to-date with the latest changes in MarkBind, which may be code-related.
+- Improving the clarity of the documentation, which may or may not be code-related.
+
+</box>
+
+**Before making any changes**, go through the following steps:
+
+1. Check that the issue has been resolved in the [latest master branch docs](https://markbind-master.netlify.app/). 
+1. Check that the issue has been raised, and if not, raise an issue.
+   1. For trivial changes (like this one), you may skip this step.
+   1. This step is important for more significant changes, as it allows the developer to propose changes and get feedback from the MarkBind team.
+
+If the issue has been triaged and is ready to be worked on, we can proceed to make the changes. Since this is a trivial (and assuming correct) change, we will not be raising an issue and will proceed to make the changes directly.
+
+1. Check out a new branch from the latest master branch.
+1. Open the `docs/userGuide/components/popups.md` file.
+1. Replace `Pop-Up`/`pop-up` with `Popup`/`popup` in the file.
+1. Update the layout in `docs/_markbind/layouts/userGuide.md` to modify the site navigation item under `Using Components` from `Pop-Ups` to `Popups`.
+1. Search for `Pop-Up`/`pop-up` in the `docs` directory to ensure that there are no other occurrences of the word. Else, replace them as well.
+
+<box type="important" light>
+
+**Checkpoint**
+
+Check that the following are true:
+- [ ] The rendered site navigation item under `Using Components` in the user guide is now `Popups`
+- [ ] The word `Pop-Up`/`pop-up` has been replaced with `Popup`/`popup` in the rendered [page](../../userGuide/components/popups.html).
+
+</box>
+
+## Make a Pull Request & Verify the PR Preview
+
+Once you have made the changes, commit them and push them to your forked repository.
+Now, you can make a pull request targeting the `master` branch of the MarkBind repository.
+
+<div id="pr-steps">
+
+1. Fill in the pull request template.
+1. Wait for the CI checks to pass.
+   1. For first-time contributors, you may need to wait for the MarkBind team to approve your PR before the CI checks can run.
+1. Verify that the PR preview is working.
+   1. The PR preview is generated by netlify/markbind-master/deploy-preview.
+   1. You can find the link to the PR preview in the checks section of the PR.
```

Comment:
I think a screenshot on where to find the PR preview link will be useful here :)